core/src/com/po/fuck/view/classdrawers/factories/HandedWeaponDrawerFactory.java

```diff
@@ -0,0 +1,40 @@
+package com.po.fuck.view.classdrawers.factories;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.math.Vector2;
+import com.po.fuck.model.weapons.Glock;
+import com.po.fuck.model.weapons.HandedWeapon;
+import com.po.fuck.model.weapons.LaserGun;
+import com.po.fuck.view.CenterDrawer;
+import com.po.fuck.view.Renderer;
+import com.po.fuck.view.classdrawers.ClassDrawer;
+
+public class HandedWeaponDrawerFactory {
+    
+    static {
+        Renderer.addDrawer(Glock.class, HandedWeaponDrawerFactory.get(Glock.class, new Sprite(new Texture("glock3.png"))));
+        Renderer.addDrawer(LaserGun.class, HandedWeaponDrawerFactory.get(LaserGun.class, new Sprite(new Texture("laser_gun2.png"))));
+    }
+
+    public static <T extends HandedWeapon> ClassDrawer<T> get(Class<T> clz, Sprite other_sprite){
```

Comment:
`other_sprite` - not java style naming, also not very obvious, maybe add javadoc or rename it